cicecore/cicedyn/general/ice_init.F90

```diff
@@ -443,7 +446,9 @@ subroutine input_data
       ktransport = 1           ! -1 = off, 1 = on
       calc_Tsfc = .true.       ! calculate surface temperature
       update_ocn_f = .false.   ! include fresh water and salt fluxes for frazil
+      cpl_frazil = 'fresh_ice_correction' ! type of coupling for frazil ice
       ustar_min = 0.005        ! minimum friction velocity for ocean heat flux (m/s)
+      hi_min = p01             ! minimum ice thickness allowed (m)
```

Comment:
Is there a rationale for a minimum of 1cm? Consider including a reasonable range in the comment

Reply:
I'm not sure I understand your comment. This is "as-is" from the Consortium. New code updates or features can be pushed back to the Consortium, but we don't modify what is in an update in our fork.